### PR TITLE
[KOGITO-1539] Renaming and removing duplicates

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

**PLEASE DO NOT MERGE ANY PRs BEFORE THIS ONE**

See:
https://issues.redhat.com/browse/KOGITO-1539

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Review by evacchi — APPROVED** [review]

File: process-timer-quarkus/README.md [hunk 1]
```diff
@@ -81,6 +81,16 @@ This needs to be given when starting process instance as delay attribute of type
 
 ## Build and run
 
+By default the [Jobs Service integration](#use-kogito-job-service-as-external-timer-service) is enabled for this example. To disabled it, just comment this dependency in the `pom.xml` file:
```
Comment (MarianMacik) [inline process-timer-quarkus/README.md]: To disable it...

File: process-timer-springboot/README.md [hunk 2]
```diff
@@ -75,12 +75,21 @@ This example shows
 * Cycle timer AfterTimer
 <p align="center"><img src="docs/images/timersCycleAfterTimerScriptTask.png"></p>
 
-
 Timer expression is expected to be given in ISO-8601 format e.g. PT30S - wait 30 seconds before expiring. 
 This needs to be given when starting process instance as delay attribute of type string. 
 
 ## Build and run
 
+By default the [Jobs Service integration](#use-kogito-job-service-as-external-timer-service) is enabled for this example. To disabled it, just comment this dependency in the `pom.xml` file:
```
Comment (MarianMacik) [inline process-timer-springboot/README.md]: Same here.

File: process-timer-springboot/pom.xml [hunk 3]
```diff
@@ -1,23 +1,19 @@
...
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.kie.kogito</groupId>
     <artifactId>kogito-examples</artifactId>
     <version>8.0.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>kogito-timer-springboot</artifactId>
-  <name>kogito-timer-springboot</name>
+  <name>Kogito Examples :: Process Timer With Springboot</name>
```
Comment (MarianMacik) [inline process-timer-springboot/pom.xml]: Spring Boot

File: process-business-rules-springboot/pom.xml [hunk 4]
```diff
@@ -9,12 +9,10 @@
     <version>8.0.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>kogito-business-rules-springboot</artifactId>
-  <name>Kogito-business-rules-springboot</name>
+  <name>Kogito Examples :: Kogito Business Rules Springboot</name>
```
Comment (MarianMacik) [inline process-business-rules-springboot/pom.xml]: Spring Boot

File: process-optaplanner-springboot/pom.xml [hunk 5]
```diff
@@ -7,9 +7,9 @@
     <version>8.0.0-SNAPSHOT</version>
   </parent>
   <groupId>org.kie.kogito.examples</groupId>
-  <artifactId>jbpm-optaplanner-springboot-example</artifactId>
+  <artifactId>process-optaplanner-springboot</artifactId>
 
-  <name>Kogito Example :: jBPM, OptaPlanner and SpringBoot</name>
+  <name>Kogito Example :: Process, OptaPlanner and SpringBoot</name>
```
Comment (MarianMacik) [inline process-optaplanner-springboot/pom.xml]: Spring Boot

**Review by MarianMacik — COMMENTED** [review]
Just typos as before. Otherwise looks OK.

File: process-timer-quarkus/README.md (hunk 1 above)
Comment (ricardozanini) [reply process-timer-quarkus/README.md]: Gosh, I forgot to apply your comments from the other PR. Shame on me. Sorry about that.

**Comment by ricardozanini** [comment]
@MarianMacik, typos resolved. I'm sorry to make you point them again.

**Comment by evacchi** [comment]
all green, we can merge as soon as @MarianMacik acks

**Review by krisv — APPROVED** [review]

**Comment by krisv** [comment]
@ricardozanini can you work with @sterobin as this requires similar updates in the documentation, chapter "example applications with kogito services".  This also means we have to sync the switch of the website with the release.

**Pull request merged**